profiles/common/modules/features/communities/communities.module

```diff
@@ -4,51 +4,83 @@
  * Code for the communities feature.
  */
 
-include_once('communities.features.inc');
+include_once 'communities.features.inc';
 
-/*
-* hook_views_pre_render
-*/
+/**
+ * Implements hook_views_pre_render().
+ */
 function communities_views_pre_render(&$view) {
   global $user;
 
   if ($view->name == 'communities_directory' || $view->name == 'communities_directory_my') {
-    // add alphabetical index
-    $alphabet = array('a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z');
+    // Add alphabetical index.
+    $alphabet = array('a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
+      'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
+    );
     $output = '';
 
     $i = 0;
     $len = count($alphabet);
     foreach ($alphabet as $letter) {
-      if (arg(1) == 'my')
+      if (arg(1) == 'my') {
         $url = arg(0) . "/my/" . $user->uid . "/" . $letter;
-      else
+      }
+      else {
         $url = arg(0) . "/" . $letter;
+      }
 
-      if ($i == 0) { // first
-        $output .= l($letter, $url, array('attributes' => array('type' => 'small', 'action_bar' => 'first', 'btn_group' => 'first', 'class' => array('hidden-phone'))));
+      if ($i == 0) {
+        // First.
+        $output .= l($letter, $url, array(
+          'attributes' => array(
+            'type' => 'small',
+            'action_bar' => 'first',
+            'btn_group' => 'first',
+            'class' => array('hidden-phone'),
+          ),
+          )
+        );
       }
```

Comment:
The indentation here is not correct, you can see from lines 40+41 that the parentheses are on the same indentation level. The code sniffer doesn't seem to detect it.

These inline arrays in function calls can be quite hard to read. It is often suggested to solve it by putting the value in a separate variable. This will make it much easier to read. For example like this:

```
$options = array(
  'attributes' => array(
    ...
  ),
);
$output .= l($letter, $url, $options);
```

This same problem appears several times in this file.